incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
Look at it the same as any other image in a chart. It is the chart maintainers duty to ensure that the image is pushed and works. Additionally CI can/will check for the functionality of the chart so would catch any missing images. 

For example [in the mariadb chart](https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/deployment.yaml#L21) the `bitnami/mariadb` image is used. The chart does not give an idea of when or how that image gets pushed or updated. 

I think aspirationally we should have a process in the CI pipeline to auto build/push images. Maybe with a convention on where images would live (images folder) and that the tag would be equivalent to the chart version.

Was there a particular concern you had?